languages/swift/exercises/concept/loops/.docs/after.md

```diff
@@ -0,0 +1,267 @@
+## Loops
```

Comment:
See my comment for the other `after.md` file.
I would keep only additional information here: additional links and the "labels" section (otherwise it is likely that this extra section is overseen by students).